api.py

```diff
@@ -6,10 +6,10 @@
 app = FastAPI()
 
 class ScoringItem(BaseModel): 
-    YearsAtCompany: float 
-    EmployeeSatisfaction: float 
-    Position:str
-    Salary: int
+    sepal_length: float 
+    sepal_width: float
+    petal_length: float 
+    petal_width: float
 
 with open('model/model.pkl', 'rb') as f: 
     model = pickle.load(f)
```

Comment:
Add exception to handle possible errors
```suggestion
    try:
        with open('model/model.pkl', 'rb') as f: 
            model = pickle.load(f)
    except Exception as e:
        logging.error(f"Error loading model: {e}")
        raise HTTPException(status_code=500, detail="Model loading error")
```

Reply:
Sure, this changes are already tested?

Reply:
This was an example of commenting and suggesting changes in several lines, but it was not tested. We would need to standardize error handling across the project. A document would be a good fit.

Reply:
This code seems to dont work. It will be adressed in the next pr